api/controller.py

```diff
@@ -952,10 +952,16 @@ def series(self, series_name):
 
         lane = SeriesLane(self._db, series_name)
         annotator = self.manager.annotator(lane)
+        facets = load_facets_from_request()
+        if isinstance(facets, ProblemDetail):
+            return facets
+        pagination = load_pagination_from_request()
+        if isinstance(pagination, ProblemDetail):
+            return pagination
```

Comment:
Hmm, I won't fix that now, but I've fixed the `load_*` parts for the contributor and recommendation feeds.